### ISSUE TITLE
ghc 8.8 / template-haskell 2.15.0: build failure

### ISSUE THREAD

**Issue by deepfire** [RyanGlScott/th-lift#38]
```
[1 of 1] Compiling Language.Haskell.TH.Lift ( src/Language/Haskell/TH/Lift.hs, dist/build/Language/Haskell/TH/Lift.o )

src/Language/Haskell/TH/Lift.hs:215:15: error:
    • Couldn't match type ‘TyVarBndr’ with ‘Type’
      Expected type: [Type]
        Actual type: [TyVarBndr]
    • In the third argument of ‘f’, namely ‘vs’
      In the expression: f dcx n vs cons
      In a case alternative:
          DatatypeInfo {datatypeContext = dcx, datatypeName = n,
                        datatypeVars = vs, datatypeCons = cons}
            -> f dcx n vs cons
    |
215 |       f dcx n vs cons
    |               ^^
```

**Comment by RyanGlScott** [comment]
I don't think the breakage here is due to GHC 8.8, but instead due to trying to use the pre-release `th-abstraction-0.3.0.0` (which isn't quite ready to be used). See also dreixel/generic-deriving#60.

**Comment by RyanGlScott** [comment]
This issue is fixed since `th-abstraction-0.3.0.0` is out and `th-lift` now builds with it.
